chrome/browser/ui/views/ime_driver/ime_driver_mus.cc

```diff
@@ -50,6 +63,16 @@ void IMEDriver::StartSession(ui::mojom::StartSessionDetailsPtr details) {
   mojo::MakeStrongBinding(
       base::MakeUnique<InputMethodBridge>(std::move(remote_client)),
       std::move(details->input_method_request));
+#elif defined(OS_LINUX) && defined(USE_OZONE) && !defined(OS_CHROMEOS)
```

Comment:
strictly speaking there is no need to check os_chromeos here, since it is already checked in line 56.